src/devices/tuya.ts

```diff
@@ -537,6 +537,23 @@ const definitions: Definition[] = [
         toZigbee: [],
         exposes: [e.gas(), e.tamper()],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0205', ['_TZ3210_up3pngle']),
+        model: 'TS0205_smoke_2',
```

Comment:
Same as https://github.com/Koenkk/zigbee-herdsman-converters/pull/7084/files#r1493791719

Reply:
Also differs in battery behavior.

Reply:
I think battery is also exposed for the other, can you merge them?

Reply:
Done, merged TS0205 smoke detector.
Expose of battery seems to be done by ```batteryPercentage()```; I see a battery percentage, but I did not add a expose:

![afbeelding](https://github.com/Koenkk/zigbee-herdsman-converters/assets/51886888/ea6f3c06-6cf8-44ff-b42e-928c8ca5c09d)


Also ```fz.ias_smoke_alarm_1``` returns more information than the devices reports, Should I create a ```fz.ias_smoke_alarm_2``` with only the return I see in the device debug in TuYA IOT ??

Reply:
> And how to remove the blue/white update arrow button, since this does not work when the device is sleeping. (it is needed and works OK to ge battery-percentage when pairing)

This is the case for all devices exposing battery percentage, you need to wake-up the device right before sending it

> Also fz.ias_smoke_alarm_1 returns more information than the device _TZ3210_up3pngle reports according TuYa IoT, Should I create a fz.ias_smoke_alarm_2 with only the returns I see in the device debug in TuYA IoT, and then split this TS0205 again??

If all the exposed information is correct, no

Reply:
OK, so it is done!